frontend/packages/dev-console/src/components/pipelines/pipeline-builder/PipelineBuilderFormEditor.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { useFormikContext } from 'formik';
+import { StackItem, TextInputTypes } from '@patternfly/react-core';
+import { InputField } from '@console/shared';
+import { PipelineParameters, PipelineResources } from '../detail-page-tabs';
+import PipelineBuilderVisualization from './PipelineBuilderVisualization';
+import {
+  PipelineBuilderFormikValues,
+  PipelineBuilderTaskGroup,
+  SelectTaskCallback,
+  UpdateTasksCallback,
+} from './types';
+
+import './PipelineBuilderForm.scss';
+
+type PipelineBuilderFormEditorProps = {
+  namespace: string;
+  hasExistingPipeline: boolean;
+  taskGroup: PipelineBuilderTaskGroup;
+  onTaskSelection: SelectTaskCallback;
+  onUpdateTasks: UpdateTasksCallback;
+};
+
+const PipelineBuilderFormEditor: React.FC<PipelineBuilderFormEditorProps> = (props) => {
+  const { namespace, hasExistingPipeline, taskGroup, onTaskSelection, onUpdateTasks } = props;
+  const { status } = useFormikContext<PipelineBuilderFormikValues>();
+
+  return (
+    <StackItem isFilled className="odc-pipeline-builder-form__content">
```

Comment:
So when adding wrapping containers, you have to be careful what the elements you're wrapping are.

This is a `StackItem`, which should be inside a `Stack` - but if you look at the rendered output, the form gets in the way:
![Screen Shot 2020-11-05 at 5 30 PM (marked up)](https://user-images.githubusercontent.com/8126518/98304158-6fb64c80-1f8d-11eb-863b-b35672e0a23a.png)

```suggestion
```

Reply:
Thanks, makes sense. Removed. I wonder if it's okay to have something that is not a `StackItem` directly under `Stack`.